### PR TITLE
Update index.html.erb.md

### PR DESCRIPTION
**Pull request by nitedog**
Pure cosmetics for editors discretion

### CODE REVIEW

File: source/page-structure/index.html.erb.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ wcag_success_criteria:
   - 2.4.10
 ---
 
-Page structure is a useful way for all users to quickly understand how content is related. For users of assistive technology, well-structured content allows more efficient navigation and processing. Use HTML and WAI-ARIA to improve navigation and orientation on web pages and in applications.
+Well-structured content allows more efficient navigation and processing. Use HTML and WAI-ARIA to improve navigation and orientation on web pages and in applications.
```
Comment (nitedog) [inline source/page-structure/index.html.erb.md]: Seemed like a lot of filler text that does not really say much - attempt to tersify.

File: source/page-structure/index.html.erb.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ wcag_success_criteria:
...
 
 * **[Page Regions](regions.html):** Identify and mark up regions on web pages using HTML5 and WAI-ARIA roles.
 
-* **[Labeling Regions](labels.html):** Name regions to allow users to distinguish and access them.
+* **[Labeling Regions](labels.html):** Label regions to allow users to distinguish and access them.
```
Comment (nitedog) [inline source/page-structure/index.html.erb.md]: label vs name? we use label in the next sentence. kind of confusing IMO.

File: source/page-structure/index.html.erb.md [hunk 3]
```diff
@@ -27,9 +27,9 @@ Page structure is a useful way for all users to quickly understand how content i
 
 Well-structured pages are essential for many web users, including:
 
-* **People with cognitive and learning disabilities** will be able to easier prioritize and find content on the page.
+* **People with cognitive and learning disabilities** are more easily able to prioritize and find content on the page.
```
Comment (nitedog) [inline source/page-structure/index.html.erb.md]: most of the other sentences say "can", "have", "will" etc (present) but this one is future - "will be able to"

File: source/page-structure/index.html.erb.md [hunk 4]
```diff
@@ -27,9 +27,9 @@ Page structure is a useful way for all users to quickly understand how content i
...
 * **People using screen readers** can skip to the main content directly and navigate to sections that are important to them.
-* **Keyboard users** will be more efficient when they are able to navigate the page bypassing some sections. If this is not possible, users need to use multiple strokes of the tab key to navigate through the links in a specific section.
+* **Keyboard users** can navigate pages and their sections more efficiently. Otherwise users need to use multiple strokes of the tab key to navigate through all links in each section.
```
Comment (nitedog) [inline source/page-structure/index.html.erb.md]: attempt to simplify

**Comment by yatil** [comment]
Thanks, awesome changes!

**Pull request merged**